plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -82,21 +82,18 @@ func getZoneKey(node *api.Node) string {
 // pods which match the same service selectors or RC selectors as the pod being scheduled.
 // Where zone information is included on the nodes, it favors nodes in zones with fewer existing matching pods.
 func (s *SelectorSpread) CalculateSpreadPriority(pod *api.Pod, nodeNameToInfo map[string]*schedulercache.NodeInfo, nodeLister algorithm.NodeLister) (schedulerapi.HostPriorityList, error) {
-	selectors := make([]labels.Selector, 0)
-	services, err := s.serviceLister.GetPodServices(pod)
-	if err == nil {
+	selectors := make([]labels.Selector, 0, 3)
```

Comment:
> But this doesn't really change anything...

I think the goal is to avoid allocation/copying? growing backing slice involving one allocation + len(s) copying. If 3 is enough for the most common case, then great! I do not really care if this is 3 or 6, or 8, since it just several temp bytes difference. Thanks for the explanation.